src/pages/settings/Integrations.tsx

```diff
@@ -109,6 +110,18 @@ const Integrations = () => {
             }}
             fullWidth
           />
+          <StyledSelector
+            title={translate('text_63e26d8308d03687188221a5')}
+            subtitle={translate('text_63e26d8308d03687188221a6')}
+            icon={<Avatar variant="connector">{<Oso />}</Avatar>}
+            onClick={() => {
+              window.open(
+                'https://www.osohq.com/docs/guides/model-your-apps-authz#entitlements',
```

Comment:
This URL should be in the `externalUrls.ts` file

Reply:
Done ! For this one and another in the same page 🙌